### DISCUSSION TITLE
Is there a way, or are there plans to include support for KDE Plasma?

### DISCUSSION THREAD

**Discussion by wyccad** [Schneegans/Burn-My-Windows#56] [Ideas]
KDE Plasma support would be much appreciated.  There is a means to add new window effects, that takes you to the KDE store but the selection there is pretty limited.

![cropped](https://user-images.githubusercontent.com/73924876/149954735-5d17dc5b-c237-414f-943c-34b8fd0cecf7.png)

**Comment by Schneegans** [comment]
Hi there! I myself will not work on this in the foreseeable future. And this is definitely nothing which can be done from one code base (I am pretty sure that KDE's effects are not written in JavaScript). So this would rather mean creating new effects for KDE in a completely different project and re-using some of the shaders of Burn-My-Windows.

If anyone is willing to do this, go ahead! It's all available under the GPL license... And I will happily explain the code if questions arise :)

**Reply by wolfyrion** [reply]
I dont know if this is possible or not to port on KDE but if you make a kickstarter or a fund project in order to do this , I will happily donate.

**Comment by Schneegans** [comment]
A small teaser: https://www.reddit.com/r/kde/comments/va3xyw/a_small_teaser/ :wink:

**Reply by wolfyrion** [reply]
Donation done , thanks! :+1: 

Now I want more effects and more minimize/maximize effects 
keep up the good work !
Thanks again! :)

**Reply by Schneegans** [reply]
Thanks a ton! If you don't mind, I would add your user name to the list of sponsors?

**Reply by wolfyrion** [reply]
is ok , I dont mind , thanks :)

**Comment by Schneegans** [comment]
An initial version is available here: https://store.kde.org/p/1861183
